core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -53,42 +50,33 @@ angular.module('oppia').directive('aboutPage', [
             'Robyn Choo', 'Tricia Ngoon',
             'Vikrant Nanda', 'Vinamrata Singal',
             'Yarin Feigenbaum'];
+          // Define constants
+          ctrl.TAB_ID_ABOUT = 'about';
+          ctrl.TAB_ID_FOUNDATION = 'foundation';
+          ctrl.TAB_ID_CREDITS = 'credits';
+          ctrl.TAB_ID_LICENSE = 'license';
 
-          const activateTab = function(tabName: string) {
-            $("a[id='" + tabName + "']").parent().addClass(
-              activeTabClass
-            ).siblings().removeClass(activeTabClass);
-            $('.' + tabName).addClass(visibleContent).siblings().removeClass(
-              visibleContent
-            );
-          };
+          const ALLOWED_TABS = [
+            ctrl.TAB_ID_ABOUT, ctrl.TAB_ID_FOUNDATION, ctrl.TAB_ID_CREDITS];
+
+          ctrl.activeTabName = ctrl.TAB_ID_ABOUT;
 
           ctrl.onTabClick = function(tabName: string) {
             // Update hash
             WindowRef.nativeWindow.location.hash = '#' + tabName;
-            activateTab(tabName);
+            ctrl.activeTabName = tabName;
           };
           ctrl.getStaticImageUrl = function(imagePath: string) {
             return UrlInterpolationService.getStaticImageUrl(imagePath);
           };
           ctrl.$onInit = function() {
-            // Define constants
-            ctrl.TAB_ID_ABOUT = 'about';
-            ctrl.TAB_ID_FOUNDATION = 'foundation';
-            ctrl.TAB_ID_CREDITS = 'credits';
-
-            var hash = WindowRef.nativeWindow.location.hash.slice(1);
-            if (hash === ctrl.TAB_ID_FOUNDATION || hash === 'license') {
-              activateTab(ctrl.TAB_ID_FOUNDATION);
+            const hash = WindowRef.nativeWindow.location.hash.slice(1);
+            if (hash === ctrl.TAB_ID_LICENSE) {
+              ctrl.activeTabName = ctrl.TAB_ID_FOUNDATION;
```

Comment:
Do we need to move the focus to `License` or it will happen anyway? (Make sure to test this.)

Reply:
I don't think so. When license is focused, the active tab name will always be foundation.

Reply:
To be honest, I don't know the expected behavior in this case. Can you help me out here @seanlip? I'm confused

Reply:
Yes, the correct section should be focused on in this case.

Reply:
I just got confused!

> hmmm.. I don't think we should change the tab if someone edits the link, I don't see this behaviour in other pages like exploration editor pages

There is no way to go to license link by just doing a click, the only way is to changing it in the url.

Reply:
This quote is from the thread below

Reply:
@marianazangrossi Thanks for checking:

Can you please add `href="#license" in[ this anchor tag ](https://github.com/oppia/oppia/pull/9104/files#diff-8c66287c207757ecd004b456310aa006R83) and try?

(We had this anchor tag for the click, I don't know why it doesn't have the href attribute.)

Reply:
I think you refer to line 83 of about-page.directive.html file, right? It worked, I'm going to push it.

Reply:
Hey @DubeySandeep the href didn't work. Actually, there is a bug on focusing the license thing.

When I type `/about#license` when the current hash is `#foundation`, the focus happens. However, when hash is empty, `#about` or `#credits`, the focus doesn't occur. This behavior is because the anchor element is only visible on foundation section. I've tried to fix it but I didn't found any solution until now.